### PR TITLE
Bug 1867792: Allow prune to move on with Removed registry

### PR DESCRIPTION
**Pull request by ricardomaraschini**
Pruner attempts to derive internal image registry host from the last
unmanaged image or from the last image stream created. In some cases
OpenShift may not contain any of those (for instance in a fresh install
with Removed registry), this makes pruner to fail even with the flag
--prune-registry set to false.

This commit allows the pruner job to move on with the --prune-registry
flag set to false even when we can't derive the internal registry host.

### CODE REVIEW

File: pkg/cli/admin/prune/images/images_test.go [hunk 1]
```diff
@@ -39,6 +39,52 @@ import (
 
 var logLevel = flag.Int("loglevel", 0, "")
 
+func TestPruneWithoutImagesWithRegistryAccess(t *testing.T) {
+	kFake := fakekubernetes.NewSimpleClientset()
+	imageFake := &fakeimagev1client.FakeImageV1{
```
Comment (dmage) [inline pkg/cli/admin/prune/images/images_test.go:44]: Why does image client have its own variable, while apps client and build client are created inline?

Comment (ricardomaraschini) [reply pkg/cli/admin/prune/images/images_test.go]: Because `copy&paste` reasons. Fixed, thanks.

**Review by soltysh — APPROVED** [review]
/approve

**Pull request merged**